app/components/forms/events/view/edit-speaker.js

```diff
@@ -0,0 +1,62 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { protocolLessValidUrlPattern } from 'open-event-frontend/utils/validators';
+
+export default Component.extend(FormMixin, {
+
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        email: {
+          identifier : 'email',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.get('l10n').t('Please enter your email ID')
+            },
+            {
+              type   : 'email',
+              prompt : this.get('l10n').t('Please enter a valid email ID')
+            }
+          ]
+        },
+        twitter: {
+          identifier : 'twitter',
+          optional   : true,
+          rules      : [
+            {
+              type   : 'containsExactly[twitter.com]',
```

Comment:
Do not just check for twitter.com. Please validate if the url also contains a username.

Eg.

```
twitter.com - Should be marked as invalid
twitter.com/niranjan94 - should be marked as valid
```